cypress.config.js

```diff
@@ -14,6 +14,21 @@ module.exports = defineConfig({
       on('task', { removeDirectory });
       // https://github.com/bahmutov/cypress-failed-log
       require('cypress-failed-log/on')(on);
+
+      on('before:browser:launch', (browser, launchOptions) => {
+        if (browser.family === 'chromium' && browser.name !== 'electron') {
+          // Set pointer type to fine so that date inputs work properly
```

Comment:
Just wondering what the actual issue was that these launch options fix? Perhaps the comment could be updated to explain it?

Reply:
It makes the datepickers editible as cypress doesn't allow for mocking media queries - I stole the code from DGW: (https://github.com/ral-facilities/datagateway/blob/develop/packages/datagateway-dataview/cypress.config.ts) but it's similar to the launch options in playwright config. I can add a link to the testing caveats like we do in the PW config